metaserver/src/main/scala/scala/meta/languageserver/Compiler.scala

```diff
@@ -1,54 +1,52 @@
 package scala.meta.languageserver
 
 import java.io.File
+import java.io.PrintStream
 import java.nio.file.Files
 import java.util.Properties
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentHashMap
```

Comment:
What caused this to be repeated 3 times?  Some automated tool?

Reply:
Fixed with OrganizeImports